ui/src/app/shared/api/task.service.ts

```diff
@@ -44,14 +44,14 @@ export class TaskService {
       params = params.append('sort', `${sort},${order}`);
     }
     return this.httpClient
-      .get<any>(UrlUtilities.calculateBaseApiUrl() + 'tasks/definitions', {headers, params})
+      .get<any>(UrlUtilities.calculateBaseApiUrl() + 'tasks/executions', {headers, params})
```

Comment:
Why these changes? I was only expecting `tasks/executions` in `getExecutions` to become `tasks/thinexecutions`